tests/EvmAcceptanceTests/test/rpc/eth_getUncleByBlockHashAndIndex.js

```diff
@@ -0,0 +1,28 @@
+const helper = require('../../helper/GeneralHelper');
+assert = require('chai').assert;
+
+const METHOD = 'eth_getUncleByBlockHashAndIndex';
+
+describe("Calling " + METHOD, function () {
+  describe("When on Zilliqa network", async function () {
+    before(async function () {
+      if (!helper.isZilliqaNetworkSelected()) {
+        this.skip();
+      }
+    })
+
+
+    it("should return the uncle Id by block hash", async function () {
+
```

Comment:
if these kind of empty lines at the beginning of the blocks are not on purpose, please remove them as well.